src/ledger/meta.zig

```diff
@@ -145,6 +145,8 @@ pub const ConnectedFlags = BitFlags(enum(u8) {
     // 2) S's parent is connected AND S is full (S's complete block present)
     //
     // 1) is a straightfoward case, roots are finalized blocks on the main fork
+    // // QUESTION: How come there can be many roots?
```

Comment:
if a slot is rooted then its parent is also rooted

ie with chain A -> B -> C -- A and B are rooted but C still doesnt have enough votes to be considered rooted